### PR TITLE
feat: add variable lazysizes to add ability use default way

### LINKED ISSUE #18: FroshLazySizes is incompatible with Spno6NewsletterBusiness Plugin
### Please describe the feature you would like to see implemented.

Hey, ich bin der Entwickler des Newsletter Plugins aus Shopware 6. Wir haben ein Support Ticket erhalten von einem Kunden der sowohl unser Plugin als auch euer Plugin verwendet. Dabei ist aufgefallen, dass die Bilder im Newsletter nicht mehr laden aufgrund der lazy Funktion. Gibt es eine Möglichkeit das Plugin wahlweise zu deaktivieren? Sei es durch parameter oder sonstiges.

Beste Grüße
Jan Dommasch

Hey, I'm the developer of the newsletter plugin for Shopware 6. We received a support ticket from a customer who uses both our plugin and your plugin. It was noticed that the images in the newsletter no longer load due to the lazy function. Is there a way to optionally deactivate the plugin? Be it through parameters or something else.

Best regards
Jan Dommasch

### PR DESCRIPTION
**Pull request by tinect**
Closes #18

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,9 @@ https://github.com/FriendsOfShopware/FroshLazySizes/releases/latest/download/Fro
 - then run the command `bin/console media:generate-thumbnails` on the console to update the thumbnails for all images in the database
 - Clear shop cache
 
+## disable lazysizes for specific images
+Just set variable `lazysize` to `false` before using `sw_thumbnails` in the template.
```
Comment (JD297) [inline README.md:31]: lazysize => lazysizes

File: src/Resources/views/storefront/utilities/thumbnail.html.twig [hunk 2]
```diff
@@ -1,69 +1,11 @@
...
-        {% endif %}
-    {% endif %}
-
-    {% set ratio = metaProportion.width|default(1) / metaProportion.height|default(1) %}
-
-    {% set inlineStyle = "aspect-ratio:" ~ ratio ~ ";" %}
-    {% if attributes.style %}
-        {% set attributes = attributes|merge({'style': inlineStyle ~ attributes.style}) %}
+    {% if lazysizes %}
+        {% sw_include '@FroshLazySizes/storefront/utilities/lazy-thumbnail.html.twig' %}
```
Comment (JD297) [inline src/Resources/views/storefront/utilities/thumbnail.html.twig:7]: {% sw_include '@FroshLazySizes/storefront/utilities/thumbnail-lazysize.html.twig' %}

**Review by JD297 — COMMENTED** [review]
With this changes the commit works for our newsletter plugins. Thx.

File: src/Resources/views/storefront/utilities/thumbnail.html.twig (hunk 2 above)
Comment (tinect) [reply src/Resources/views/storefront/utilities/thumbnail.html.twig:7]: thanks again

**Pull request merged**